CTP/shell/src/com/navercorp/cubridqa/shell/result/FeedbackFile.java

```diff
@@ -84,30 +98,76 @@ public void onTaskContinueEvent() {
 
 	@Override
 	public void onTaskStopEvent() {
+		showTestResult();
+		
 		long taskStopTime = System.currentTimeMillis();
 		println("[TEST STOP] Current Time is " + new Date(), "Elapse Time:" + ((taskStopTime - this.taskStartTime)));
 		feedbackLog.close();
 	}
 
 	@Override
 	public void setTotalTestCase(int tbdNum, int macroSkippedNum, int tempSkippedNum) {
+		this.totalCaseNum = tbdNum;
+		this.totalSkipNum = macroSkippedNum + tempSkippedNum;
 		println("The Number of Test Cases: " + tbdNum + " (macro skipped: " + macroSkippedNum + ", bug skipped: " + tempSkippedNum + ")");
 	}
+	
+	public void showTestResult(){
+		System.out.println("============= PRINT SUMMARY ==================");
+		try {
+			Properties prop = CommonUtils.getProperties(this.statusLogName);
+			Iterator itr = prop.entrySet().iterator();
+			while (itr.hasNext()){
+	            Entry e = (Entry)itr.next();
+	            System.out.println(e.getKey() + ": " + e.getValue());
+	        }
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+		
+	}
+	
+	private synchronized void updateTestingStatistics(){
+		try {
+			Properties prop = com.navercorp.cubridqa.common.CommonUtils.getProperties(this.statusLogName);
+			prop.setProperty("total_case_count", String.valueOf(this.totalCaseNum));
+			prop.setProperty("total_executed_case_count", String.valueOf(this.totalExecutedCaseNum));
+			prop.setProperty("total_success_case_count", String.valueOf(this.totalSuccNum));
+			prop.setProperty("total_fail_case_count", String.valueOf(this.totalFailNum));
+			prop.setProperty("total_skip_case_count", String.valueOf(this.totalSkipNum));
+			
+			com.navercorp.cubridqa.common.CommonUtils.writeProperties(this.statusLogName, prop);
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+	}
 
 	@Override
 	public void onTestCaseStopEvent(String testCase, boolean flag, long elapseTime, String resultCont, String envIdentify, boolean isTimeOut, boolean hasCore, String skippedType, int retryCount) {
 		String head;
 		if (skippedType.equals(Constants.SKIP_TYPE_NO)) {
 			head = flag ? "[OK]: " : "[NOK]: " + Constants.RETRY_FLAG + " = " + retryCount;	
+			if (retryCount != 1) {
+				if (flag) {
+					this.totalSuccNum++;
+				} else {
+					this.totalFailNum++;
+				}
+			}
 		} else if (skippedType.equals(Constants.SKIP_TYPE_BY_MACRO)) {
 			head = "[SKIP_BY_MACRO]";
+			this.totalSkipNum++;
```

Comment:
Below can be removed because it is set in onTestCaseStopEvent.
this.totalSkipNum++;